db_store/src/meta.rs

```diff
@@ -0,0 +1,38 @@
+use std::str::FromStr;
```

Comment:
this all makes sense but it seems like it could get confusing that the `MetaValue` struct is defined in the `lib.rs` module and this more generic accessor function module is named `meta.rs`? i know these are dealing with just setting and getting key/value pairs but maybe these two modules can be unified in some way?

Reply:
I agree it is confusing and I can drop the simpler `meta` if we want, but I did change MetaValue to use `meta` under the covers.  48754ee

Reply:
i think having the metavalue's use the meta module functions is good